src/client.py

```diff
@@ -45,32 +51,46 @@ def _login(email: str, password: str):
                 raise IncorrectCredentials
         return session
 
-    def get_classes(self, target_day: datetime):
+    def get_classes(self, target_day: datetime, family_id: str):
         response = self.session.get(
             classes_endpoint(self.box_name),
             params={
                 "box": self.box_id,
                 "day": target_day.strftime("%Y%m%d"),
-                "familyId": "",
+                "familyId": family_id,
+                "_": time.time(),
             },
         )
-        return response.json().get("bookings")
+        if response.status_code == HTTPStatus.OK:
+            bookings = response.json().get("bookings")
+            self.logger.info(f"Retrieved {len(bookings)} classes for day {target_day.strftime('%A, %Y-%m-%d')}")
+            return bookings
+        else:
+            self.logger.error("Error getting classes for the day %s on request %s with response: %s %s %s",
+                              target_day.strftime("%Y%m%d"), response.url, response.status_code, response.reason, response.text)
+            raise ErrorResponse
 
-    def book_class(self, target_day: datetime, class_id: str) -> bool:
+    def book_class(self, target_day: datetime, class_id: str, family_id: str) -> bool:
         response = self.session.post(
             book_endpoint(self.box_name),
             data={
                 "id": class_id,
                 "day": target_day.strftime("%Y%m%d"),
                 "insist": 0,
-                "familyId": "",
+                "familyId": family_id,
```

Comment:
I am fine with this, even though I cannot really test in real life, but I don't see any unit test being modified which doesn't smell good

Reply:
I've tested in real world but you're right, I'll try and add some tests for this new parameter.